.ruby-version

```diff
@@ -1 +1 @@
-2.2.5
+2.3.1
```

Comment:
remove this.

Reply:
Remove this?

Reply:
Sorry, I don't want to remove the file, but I do want to revert the changes. This was a local development thing that shouldn't be committed

Reply:
Ah gotcha.